psignifit/psigniplot.py

```diff
@@ -213,10 +218,15 @@ def plot_prior(result: Result,
                line_color: Union[str, List[float], np.ndarray] = '#0069AA',  # blue
                line_width: float = 2,
                marker_size: float = 30):
-    """ Plot the priors on the different parameters.
-
-    The coloured psychometric functions correspond to the 0%, 25%, 75% and 100%
-    quantiles of the prior.
+    """ Plot the priors for the threshold, width and lambda parameters.
+    
+    The upper pannels show the priors. The lower pannels show a set of psychometric 
```

Comment:
"pannels" -> "panels" (3x)
(a pannel is apparently the stomach of a hawk)

Reply:
fixed